submissions/pashapushak1996/html-css-popup/index.html

```diff
@@ -0,0 +1,218 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link
+      href="https://fonts.googleapis.com/css2?family=Uchen&display=swap"
+      rel="stylesheet" />
+  <link rel="stylesheet" href="css/main.css" />
+  <title>Popup</title>
+</head>
+<body>
+<div class="wrapper">
+  <header class="header container">
+    <nav class="header__menu">
+      <ul class="header__list">
+        <li>
+          <a href="#" class="header__link">Mail</a>
+        </li>
+        <li>
+          <a href="#" class="header__link">Images</a>
+        </li>
+        <li>
+          <input type="checkbox" id="app-icon" name="app-icon" />
+          <label for="app-icon" class="app-icon">
+            <img src="icons/app-icon.png" alt="" />
+          </label>
+          <div class="popup">
+            <div class="popup__inner">
+              <ul class="popup__items">
+                <li>
+                  <a href="#" class="popup__item">
+                    <div class="popup__image">
+                      <img
+                          src="icons/calendar-icon.png"
+                          alt="calendar-icon" />
+                    </div>
+                    <p>Calendar</p>
```

Comment:
I missed it earlier, better not to use `p` inside `a`.
https://caninclude.glitch.me/caninclude?child=p&parent=a